src/components/atoms/MessageHeader.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <div class="flex border-b border-gray-200 p-2">
+        <img :src="'../src/assets/' + imagePath + imageName" :alt="altText" class="h-10 mt-auto w-10">
+        <span class=" font-bold pt-2 pl-2">{{ headerText }}</span>
+    </div> 
+</template>
+
+<script>
+export default {
+    props: {
+        imagePath: String,
```

Comment:
Do we need the image path prop? We can combine the image path and image name into one prop.

Reply:
Oops that's my bad, I didn't want a long alt-text name so I separated the two. Will fix.